src/frontend/typer/oppgave.ts

```diff
@@ -7,6 +7,22 @@ export interface IDataForManuellJournalføring {
     person?: IPerson;
 }
 
+export interface IFinnOppgaveRequest {
+    behandlingstema: string | undefined;
```

Comment:
```suggestion
    behandlingstema?: string;
```

I prefer using optional like this. I also think it is more correct and we can utilize typescript better.